ios/Classes/TiMapCustomAnnotationView.m

```diff
@@ -36,7 +36,7 @@ - (void)setProxy:(TiViewProxy*)customView
 
 - (void)initWithProxy:(TiViewProxy*)customView
 {
-    theProxy = [customView retain];
+    theProxy = customView;
```

Comment:
Why would you not want to retain it? The only thing I could think of is that annotations are created and removed dynamically, but we really need to test this regarding many use-cases.